packages/keystone/index.js

```diff
@@ -1 +1,15 @@
-exports.Keystone = require('./Keystone');
+const Keystone = require('./lib/Keystone');
+const PasswordAuthStrategy = require('./lib/auth/Password');
+const TwitterAuthStrategy = require('./lib/auth/Twitter');
+const { FacebookAuthStrategy } = require('./lib/auth/Facebook');
+const { BaseKeystoneAdapter, BaseListAdapter, BaseFieldAdapter } = require('./lib/adapters');
+
+module.exports = {
+  Keystone,
+  PasswordAuthStrategy,
+  TwitterAuthStrategy,
+  FacebookAuthStrategy,
```

Comment:
I'm not a fan of exporting the auth strategies from `@keystone-alpha/keystone`, could we move them to a separate package?

Reply:
Yep, 100% agree that they should be moved to a different package, just not in this PR. For now I want to make them explicit exports of this package, which is an improvement on the existing implicit exports from internal modules.